docs/contributing.rst

```diff
@@ -16,93 +16,125 @@ To contribute to the cosmos project:
 #. Link your issue to the pull request
 #. Once developments are complete on your feature branch, request a review and it will be merged once approved.
 
+Setup local development on host machine
+---------------------------------------
 
-Using Hatch for local development
----------------------------------
+This guide will setup astronomer development on host machine, first clone the ``astronomer-cosmos`` repo and enter the repo directory:
 
-We currently use `hatch <https://github.com/pypa/hatch>`_ for building and distributing ``astronomer-cosmos``.
+.. code-block:: bash
 
-The tool can also be used for local development. The `pyproject.toml <https://github.com/astronomer/astronomer-cosmos/blob/main/pyproject.toml>`_ file currently defines a matrix of supported versions of Python and Airflow for which a user can run the tests against.
+    git clone https://github.com/astronomer/astronomer-cosmos.git
+    cd astronomer-cosmos/
 
-For instance, to run the tests using Python 3.10 and Apache Airflow 2.5, use the following:
+Then install ``airflow`` and ``astronomer-cosmos`` using python-venv:
 
 .. code-block:: bash
 
-    hatch run tests.py3.10-2.5:test-cov
+    python3 -venv m env && source env/bin/activate
+    pip3 install apache-airflow
+    pip3 install -e .[dbt-postgres,dbt-databricks]
 
-It is also possible to run the tests using all the matrix combinations, by using:
+Set airflow home to the ``dev/`` directory and disabled loading example DAGs:
 
 .. code-block:: bash
 
-    hatch run tests:test-cov
+    export AIRFLOW_HOME=$(pwd)/dev/
+    export AIRFLOW__CORE__LOAD_EXAMPLES=false
 
-The integration tests rely on Postgres. It is possible to host Postgres by using Docker, for example:
+Then, initialize the database and create a new user, the command below will create a new user admin with password admin:
+
+    By default airflow will use sqlite as database, you can overwrite this by set variable ``AIRFLOW__DATABASE__SQL_ALCHEMY_CONN`` to the sql connection string.
 
 .. code-block:: bash
 
-    docker run --name postgres -p 5432:5432 -p 5433:5433 -e POSTGRES_PASSWORD=postgres postgres
+    airflow db init
 
-To run the integration tests for the first time, use:
+    airflow users create \
+        --email admin@admin.com --firstname admin \
+        --lastname admin --password admin \
+        --role Admin --username admin
+
+Run the following airflow component on the separate terminal
 
 .. code-block:: bash
 
-    export AIRFLOW_HOME=`pwd`
-    export AIRFLOW_CONN_AIRFLOW_DB=postgres://postgres:postgres@0.0.0.0:5432/postgres
-    hatch run tests.py3.8-2.5:test-integration-setup
-    hatch run tests.py3.8-2.5:test-integration
+    airflow webserver
+    airflow scheduler
+    airflow triggerer
```

Comment:
WDYT of recommending `airflow standalone`, it should run all of this? There would be a minor difference on the `admin` password, but it may be worth it.

Reply:
I think `airlfow standalone` will simplify all of that step. I'll update on the next commit

Anyway, i forgot to add note in the last commit, There's an error while parsing dags `example_cosmos_sources.py` because that dags will try to run dbt and cannot find adapter for sqlite (probably because the setup is use sqlite)
![image](https://github.com/astronomer/astronomer-cosmos/assets/22138274/abe3dc7a-6aab-4ab0-a619-df2af908ba5f)

Actually there're 4 DAGs import errors, but i think it's out of scope from this PR because it's related to setup env var for databrick, etc. WDYT about it, should i add some note on the guide?

Reply:
Hii @tatiana, i've push new changes for fix typo and change to airflow standalone mode